### ISSUE TITLE
Importing wakeypy nukes logging config

### ISSUE THREAD

**Issue by reidswanson** [fohrloop/wakepy#343]
wakeypy > 0.7.2 nukes python logging

```python

import logging
import wakepy


def main():
    log = logging.getLogger("main")
    log.info("Running in main")


if __name__ == '__main__':
    logging.basicConfig(level=logging.DEBUG)
    main()
```
When run with wakepy < 0.8.0 the program logs "Running in main" as expected. However, for wakepy > 0.7.2 nothing is logged.

I've tested on both Linux and Window with the same results.

**Comment by fohrloop** [comment]
Oh wow that should not happen! It seems that importing wakepy >0.7.2 changes logging level to `WARNING`, so everything logged at info and debug levels is lost.

**Comment by fohrloop** [comment]
It was due to some part of the code calling `logging.debug()` instead if `logger.debug()`. Fixed the issue in wakepy 0.9.1 which I've just released. Thanks for reporting it @reidswanson !
